src/app/core/module.ts

```diff
@@ -28,6 +28,7 @@ import {EndOfLifeService} from '@core/services/eol';
 import {GlobalModule} from '@core/services/global/module';
 import {HistoryService} from '@core/services/history';
 import {LabelService} from '@core/services/label';
+import {MeteringService} from '@core/services/metering';
```

Comment:
I'm wondering if we also want to keep the `MeteringService` inside the `dynamic/enterprise` folder, as it is an ee-only feature?
On the other hand for sure it is looking cleaner to have all services in one place. 🤔 

Just a thought 🤔

Reply:
Ye, this is a good idea. I'll move it.